e2e/tests/recordings.spec.ts

```diff
@@ -111,7 +111,7 @@ test.describe('call recordings, transcriptions, live-captions', () => {
         const resp = await request.get(`${baseURL}${src}`);
         expect(resp.status()).toEqual(200);
         const transcriptionData = await resp.body();
-        await expect(transcriptionData.toString()).toContain('This is a test transcription sample');
+        await expect(transcriptionData.toString().toLowerCase()).toContain('this is a test transcription sample');
```

Comment:
Imagine when it will start to hallucinate. That's like the ultimate flaky test.

Reply:
Haha 😂  😭